src/Uno.UI.Foldable/Uno.UI.Foldable.csproj

```diff
@@ -38,7 +38,7 @@
 
 	<ItemGroup>
 		<PackageReference Include="Uno.SourceGenerationTasks" />
-		<PackageReference Include="Xamarin.AndroidX.Window.WindowJava" Version="1.0.0.3-beta02" />
+		<PackageReference Include="Xamarin.AndroidX.Window.WindowJava" />
```

Comment:
Same here about android versions.

Reply:
This one is referenced unconditionally here:

https://github.com/unoplatform/uno/blob/470dc159580661a940005bbc7029a3a489405ec9/src/Uno.UI.Foldable/Uno.UI.Foldable.csproj#L39-L42

So it needs an unconditional `Update` as well.